lib/models/case.js

```diff
@@ -135,6 +137,14 @@ class Case {
 
   static find(id) {
     return this.Model.query().findById(id)
+      .eager('[activityLog]')
+      .modifyEager('[activityLog]', builder => {
+        // we only care about logs of status changes
+        builder.where('eventName', 'like', 'status%')
+          // ignore auto-forwarded status changes
+          .andWhere('eventName', '!=', 'status:with-ntco:ntco-endorsed')
```

Comment:
Can we do that filtering at render-time? It's _super_ implementation specific, so doesn't really have any place in framework level code.

Reply:
Done.